tests/calculators/test_vasp.py

```diff
@@ -250,3 +251,43 @@ def test_vasp_scratchdir(tmp_path, monkeypatch):
     assert 'TEST_energy' in ats[0].info
     assert 'TEST_forces' in ats[0].arrays
     # ase.io.write(sys.stdout, list(configs_eval), format='extxyz')
+
+
+
+def test_vasp_per_configuration(tmp_path):
+    vasp_kwargs = {
+        "encut": 200.0,  # kinetic energy cutoff
+        "ediff": 1.0e-3,
+        "kspacing": 1.0,
+        "pp": os.environ['PYTEST_VASP_POTCAR_DIR'],
+    }
+    
+    atoms = [Atoms('Si', cell=(2, 2, 2), pbc=[True] * 3), Atoms('Si', cell=(2, 2, 2), pbc=[True] * 3), Atoms('Si', cell=(2, 2, 2), pbc=[True] * 3)]
+
+    tmp = copy.deepcopy(vasp_kwargs)
+    tmp['encut'] = 220.0
+    atoms[1].info["WFL_CALCULATOR_INITIALIZER"] = Vasp
+    atoms[1].info["WFL_CALCULATOR_KWARGS"] = tmp
+
+    tmp = copy.deepcopy(vasp_kwargs)
+    tmp['encut'] = 240.0
+    atoms[2].info["WFL_CALCULATOR_KWARGS"] = tmp
+
+    calculator = (Vasp, [], vasp_kwargs)
+
+    configs_eval = generic.calculate(
+        inputs=ConfigSet(tmp_path / 'vasp_in.xyz'),
```

Comment:
Now that it's in this branch I can run the new vasp test locally, and it's failing, because nothing in the test actually creates `vasp_in.xyz` as far as I can tell.